calendar.html

```diff
@@ -31,7 +31,7 @@ <h3>Busy Parent Planner</h3>
         <ul>
             <button class="prev"><span>&#10094;</span></button>
             <button class="next"><span>&#10095;</span></button>
-            <li><span id="monthName">January</span><span id="year">2021</span></li>
+            <li><span id="monthName">January -- </span><span id="year">2021</span></li>
```

Comment:
was this ' -- ' intentional? the JS render overwrites this text anyway, so it won't show in the final product